src/content/docs/en/guides/upgrade-to/v5.mdx

```diff
@@ -582,3 +582,67 @@ Know a good resource for Astro v4.0? [Edit this page](https://github.com/withast
 ## Known Issues
 
 Please check [Astro's issues on GitHub](https://github.com/withastro/astro/issues/) for any reported issues, or to file an issue yourself.
+
+
+### [changed]: `entryPoint` type inside the hook `astro:build:ssr`
+In Astro v4.x, the `entryPoint` type was `RouteData`.
+
+Astro v5.0 the `entryPoint` type is `IntegrationRouteData`, which contains a subset of the `RouteData` type. The fields `isIndex` and `fallbackRoutes` were removed.
+
+#### What should I do?
+Update your adapter to change the type of `entryPoint` from `RouteData` to `IntegrationRouteData`.
+
+```diff
+-import type {RouteData} from 'astro';
++import type {IntegrationRouteData} from "astro"
+
+-function useRoute(route: RouteData) {
++function useRoute(route: IntegrationRouteData) {
+
+}
+```
+
```

Comment:
```suggestion

Please see the [API reference for IntegrationRouteData]() for full details of the fields now available.

```
Seems like this would be a good place to link to the reference so that people can check this out if they need more details!